examples/basic_operations.js

```diff
@@ -1,27 +1,42 @@
 "use strict";
 
-var Redis = require("ioredis");
-var redis = new Redis();
+const Redis = require("ioredis");
+const redis = new Redis(process.env.redisPort, process.env.redisEndpoint, {password: process.env.redisPW});
```

Comment:
We should recommend passing a single option since it's easier for people to understand:

```javascript
new Redis({
  port: process.env.redisPort,
  host: process.env.redisEndpoint,
  password: process.env.redisPW
});
```